documentation/modules/auxiliary/scanner/http/apache_mod_cgi_bash_env.md

```diff
@@ -0,0 +1,94 @@
+## Vulnerable Application
+
+This module scans for the Shellshock vulnerability, a flaw in how the Bash shell handles external environment variables. This module targets CGI scripts in the Apache web server by setting the HTTP_USER_AGENT environment variable to a malicious function definition.
+
+## Verification Steps
+1. Do: run `msfconsole`
+2. Do: set `RHOSTS [IP]`
+3. Do: set `TARGETURI [URI]`
+4. Do: `run`
+
+### To check if a host is vulnerable to the attack
+1. run (on the host)
+```
+env 'x=() { :;}; echo vulnerable' 'BASH_FUNC_x()=() { :;}; echo vulnerable' bash -c "echo test"
+```
+2. The shell will return the below text if the environment is vulnerable
+``` 
+vulnerable
+bash: BASH_FUNC_x(): line 0: syntax error near unexpected token `)'   
+bash: BASH_FUNC_x(): line 0: `BASH_FUNC_x() () { :;}; echo vulnerable'
+bash: error importing function definition for `BASH_FUNC_x'
+test
+``` 
+
+## Options
```

Comment:
This has a different format.  Instead of a number list, its h3 (###) for each option, and the description below it.
You can remove some default ones that are on most modules such as Proxies, RHosts, rport, ssl, targeturi, threads, and vhost.  Those default settings, the value can be wrapped in ticks (`)